src/rgw/rgw_op.cc

```diff
@@ -492,7 +492,7 @@ int rgw_build_bucket_policies(RGWRados* store, struct req_state* s)
     map<string, bufferlist> uattrs;
 
     ret = rgw_get_user_attrs_by_uid(store, acct_acl_user.uid, uattrs);
-    if (!ret) {
+    if (! ret) {
```

Comment:
re: CodingStyle, the [google style guide](https://google.github.io/styleguide/cppguide.html) says:
```
// No spaces separating unary operators and their arguments.
x = -5;
++x;
if (x && !y)
```